webClient/src/app/editor/code-editor/monaco-settings/monaco-settings.component.html

```diff
@@ -41,16 +41,17 @@ <h2>Editor Settings</h2>
       </div>
     </div>
     <div class="button-row">
-      <button mat-raised-button class="menu-button" color="warn" (click)="resetToDefault()">Defaults</button>
-      <button mat-raised-button class="menu-button" color="primary" (click)="commitToConfigService()">Save</button>
+      <button mat-raised-button class="menu-button defaults-button" color="warn" (click)="resetToDefault()">Defaults</button>
+      <button mat-raised-button class="menu-button save-button" color="primary" (click)="commitToConfigService()">Save</button>
     </div>
   </div>
+  <div class="divider"></div>
   <div class="preview">
-    <h2 style="padding: 0.5em">JSON Preview</h2>
+    <h4 class="title-card">Advanced</h4>
     <div class="monaco-editor-preview" #monacoPreview></div>
     <div class="button-row">
-      <button mat-raised-button class="menu-button" (click)="resetEditor()">Revert</button>
-      <button mat-raised-button class="menu-button" (click)="updateFromPreview()">Preview</button>
+      <button mat-raised-button class="menu-button" (click)="resetEditor()">Revert Preview</button>
+      <button mat-raised-button class="menu-button" (click)="updateFromPreview()">Preview In-line</button>
```

Comment:
What happens to these when someone shrinks the window? I kept the names short due to not having great css for multiple widths

Reply:
Good point, need to make buttons smaller and maybe text wrap